integration/tests/user_scenario/basic_workflow/single_env_upgrade_forge_mod_revert_change.rb

```diff
@@ -0,0 +1,148 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-117 - C63602- Single Environment Upgrade Forge Module then Revert Change'
+
+#Init
+master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
+env_path = on(master, puppet('config print environmentpath')).stdout.rstrip
+prod_env_modules_path = File.join(env_path, 'production', 'modules')
+
+git_environments_path = '/root/environments'
+last_commit = git_last_commit(master, git_environments_path)
+
+#Verification
+motd_path = '/etc/motd'
+motd_template_path = File.join(prod_env_modules_path, 'motd', 'templates', 'motd.erb')
+motd_template_contents_forge = 'Hello!'
+motd_contents_git = 'Bonjour!'
+motd_contents_forge_regex = /\A#{motd_template_contents_forge}\n\z/
+motd_contents_git_regex = /\A#{motd_contents_git}\z/
+
+#File
+puppet_file_forge = <<-PUPPETFILE
+mod "puppetlabs/motd", '1.1.1'
+PUPPETFILE
+
+puppet_file_git = <<-PUPPETFILE
+mod "puppetlabs/motd",
+  :git => 'https://github.com/puppetlabs/puppetlabs-motd',

```

Comment:
This test is only about reverting a Forge module. You should have no reference to git at all in this test.